app/www/index.html

```diff
@@ -22,7 +22,7 @@
         </div>
         <script src="cordova.js" type="text/javascript"></script>
       -->
-  <script src="script/phaser.min.js" type="text/javascript"></script>
+  <script src="phaser/build/phaser.min.js" type="text/javascript"></script>
```

Comment:
Haste recht!

Da sind wir dann beim Thema JS-Build-Prozess (hier uglify https://www.npmjs.com/package/uglify-js).

Oder man lässt das Thema komplett und macht diesen Kompromiss hier. Ich gucke sowieso nicht in den Source (und ich glaube gesehen zu haben, dass zumindest der Google-Chrome Debugger es mittlerweile schnallt und in phase.js springt). 

Das ist aber eine sehr "Oltmannsche" Auffassung der Dinge, die wohl sonst niemand mit mir teilt :)